progress/meter.go

```diff
@@ -111,6 +111,12 @@ func (p *ProgressMeter) Start() {
 	}
 }
 
+func (p *ProgressMeter) Pause() {
+	if atomic.SwapInt32(&p.started, 0) == 1 {
```

Comment:
Nit: Using `atomic.CompareAndSwapInt32(&p.started, 1, 0)` accomplishes the same behavior.